src/Sentry/Sentry.csproj

```
@@ -21,6 +21,18 @@
   </ItemGroup>
   <!-- Ben.Demystifier -->
 
+  <!-- DiagnosticSource -->
+  <PropertyGroup Condition="!$(TargetFramework.StartsWith('netstandard')) and '$(TargetFramework)' != 'net461'">
+    <DefineConstants>$(DefineConstants);HAS_DIAGNOSTIC_INTEGRATION</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'netcoreapp3.0'">
```

Can we negate this instead? So when we add net6.0 it just falls in without having to change this?

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' != 'net461' and !$(TargetFramework.StartsWith('netstandard'))">
```